server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/ForceMergeRequest.java

```diff
@@ -111,6 +112,14 @@ public ForceMergeRequest flush(boolean flush) {
         return this;
     }
 
+    @Override
+    public String getDescription() {
+        return "Force-merge indices[" + Arrays.toString(indices()) +
```

Comment:
Arrays.toString(...) already adds brackets, so maybe remove the brackets in string around it?

Reply:
Good catch! Adjusted now.